### ISSUE TITLE
A11y: Focus Indicator for Promotion submit button has insufficient contrast

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5206]
The Focus Indicator for the Promotion submit button has insufficient contrast against the background. It must have sufficient contrast for both the button hover state AND the background.

{panel:bgColor=#fffae6}
Button should probably match the same styling as the buttons to remove products from the cart (see linked ticket).
{panel}

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-38924**



[created from NEXT-38924, comment 505551]

**Comment by shopwareBot** [comment]
[public]

 > Status Change: **In Progress**

 Work on this issue has started. It is now being implemented.

 

[created from NEXT-38924, comment 511516]

**Comment by shopwareBot** [comment]
[public]

 > Status Change: **In Review**

 Work on this ticket is complete and is now pending review.



[created from NEXT-38924, comment 512094]

**Comment by shopwareBot** [comment]
[public] 

 > FixVersion Change: **6.6.9.0**

 This issue will be / was solved in Shopware version ***6.6.9.0***.



[created from NEXT-38924, comment 515392]
